tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -672,7 +671,7 @@ Resources:
         - python3.8
       Content:
         S3Bucket: !Ref RedshiftConfigTestingBucket
-        S3Key: tools/NodeConfigCompare/boto3-redshift-serverless.zip
+        S3Key: configurations/boto3-redshift-serverless.zip
```

Comment:
I assume with these changes, you are also making changes in S3. Approving the changes